index.html

```diff
@@ -87,7 +87,35 @@ <h1 class="logo-brand" itemprop="brand">MACILE</h1>
       </div>
     </div>
   </nav>
-
+  <div class="teaser">
+    <h1 class="teaser-item__tittle teaser-item__main-title">OUR NEWS</h1>
+    <div class="teaser-item">
+      <span class="teaser-item__name teaser-item__name_pink">MACILE NEWSLETTER</span>
+      <div class="teaser-item__content">
+       <span class="teaser-item__date teaser-item__date_pink">July-October, 2019</span>
+       <h1 class="teaser-item__tittle">We are the heroes</h1>
+       <p class="teaser-item__text">"We are the heroes" is the first digital newsletter of MACILE, a non-profit organization based on learning about Math, Science, Engineering and Language.
+       </p>
+       <a class="teaser-item__botton teaser-item__botton_pink">Read this issue</a>
+      </div>
+      <div class="teaser-item__img">
+       <img src="/images/newsletter/graduation_20191026MA-original.jpg" itemprop="image">
+      </div>
+    </div>
+    <div class="teaser-item teaser-item__image-first">
+      <span class="teaser-item__name teaser-item__name_blue">MACILE ACADEMY BULLETIN</span>
+      <div class="teaser-item__content">
+       <span class="teaser-item__date teaser-item__date_blue">Summer 2007-2008</span>
+       <h1 class="teaser-item__tittle">We are the Adventurers</h1>
+       <p class="teaser-item__text">The MACILE Summer Academy is a rigorous five week program in grades 6 to 12. The curriculum includes math, science, engineering, technology and language.
+      Courses and activities are designed to stimulate curiosity, creativity and passion for learning; also to build the students' skills, analytical abilities and rational thinking. The program was piloted in july 2007 and 2008.</p>
+       <a class="teaser-item__botton teaser-item__botton_blue">Read this issue</a>
```

Comment:
`blue` and `pink` are modifiers of button. double check the classes for the `teaser-name` too.

Reply:
What do you mean hear?

Reply:
We solved this during standown. What the PR has is fine following BEM standards published by Zandex. I opened #131to address some inconsistencies I'm seeing across the code base and for us to set our specific BEM standards.